src/components/button/button.tsx

```diff
@@ -2,7 +2,7 @@ import React, { SyntheticEvent, FC } from "react"
 import { StyledButton, StyledCircularProgress } from "./styled"
 import { DEFAULT } from "./constants"
 
-type ButtonType = "default" | "noFill" | "borderless"
+type ButtonType = "default" | "hollow" | "borderless"
```

Comment:
so it's a breaking change right?

Reply:
Hmmm yes. Yet not important. Wanted to avoid the 2 word naming. Nobody is using these buttons types yet but we could publish our first major version update due to type renaming  🥇 
Else, in case we want to avoid that, it's not a big issue to revert this back.